near-sdk/src/collections/vector.rs

```diff
@@ -126,24 +118,18 @@ impl<T> Vector<T> {
         } else {
             let lookup_key = self.index_to_lookup_key(index);
             if env::storage_write(&lookup_key, &raw_element) {
-                match env::storage_get_evicted() {
-                    Some(x) => x,
-                    None => env::panic(ERR_INCONSISTENT_STATE),
-                }
+                expect_consistent_state(env::storage_get_evicted())
             } else {
                 env::panic(ERR_INCONSISTENT_STATE);
             }
         }
     }
 
     /// Iterate over raw serialized elements.
-    pub fn iter_raw<'a>(&'a self) -> impl Iterator<Item = Vec<u8>> + 'a {
+    pub fn iter_raw(&self) -> impl Iterator<Item = Vec<u8>> + '_ {
```

Comment:
Interested in hearing what's going on here, but not urgent or blocking

Reply:
ah so with any reference you can specify the lifetime explicitly, but if there is only one possible lifetime that can be used, it can be inferred instead. This just cleans up the usage so that it's more easily read. Anytime there is a `'_` it's an [anonymous lifetime](https://doc.rust-lang.org/edition-guide/rust-2018/ownership-and-lifetimes/the-anonymous-lifetime.html) that just means that lifetime is inferred.

In this case, the lifetime must come from `&self` as it's the only reference as a parameter.